Realm/RLMProperty.mm

```diff
@@ -311,4 +311,8 @@ - (BOOL)isEqualToProperty:(RLMProperty *)property {
         && (_objectClassName == property->_objectClassName  || [_objectClassName isEqualToString:property->_objectClassName]);
 }
 
+- (NSString *)description {
+    return [NSString stringWithFormat:@"%@ {\n\ttype = %@\n\tobjectClassName = %@\n\tindexed = %@\n\tisPrimary = %@\n}", self.name, RLMTypeToString(self.type), self.objectClassName, self.indexed ? @"YES" : @"NO", self.isPrimary ? @"YES" : @"NO"];
```

Comment:
[`RLMObjectBase`'s description](https://github.com/realm/realm-cocoa/blob/067fbdbaf8dde1d725cbd1b6c8a74b7bb8c1e4bc/Realm/RLMObjectBase.mm#L225) terminates lines with a semicolon for consistency with typical descriptions in Foundation. Could you do the same here?